ui/bundle_vendor.js

```diff
@@ -127,6 +127,16 @@ require('leaflet-search');
 window.L = require('leaflet');
 window.leafletPip = require('@mapbox/leaflet-pip');
 
+delete L.Icon.Default.prototype._getIconUrl;
+L.Icon.Default.mergeOptions({
+    iconRetinaUrl: 'https://assets.xibo-adspace.com/map/marker-icon-2x.png',
```

Comment:
@dasgarner , can you suggest what should be the url for these map icons?

Reply:
@maurofmferrao, Can we go through this in the morning please?

Reply:
Yes, I think we should download and include the icons in the `web/theme/default` folder (maybe in a `map` sub folder).

I think you might need Mauro's help to get the right URL into the vendor bundle though - we have to set something to handle installs in sub-folders.

Reply:
@dasgarner I suggested to have those assets in `ui/src/assets/map` instead, so we can use them in both `twig` and `js` side easily